src/FilterInterface.php

```diff
@@ -4,5 +4,5 @@
 
 interface FilterInterface
 {
-    public function shouldSend($payload, $accessToken);
+    public function shouldSend($payload, string $accessToken);
```

Comment:
Other instances of $payload are type `EncodedPayload`. Does that also apply here?

Reply:
Yes, it does! I've got a mess of these changes queued up and am batching them for ease of consumption. The next bundle is https://github.com/rollbar/rollbar-php/pull/525.

Reply:
:+1: Sounds good.